### ISSUE TITLE
Higher "cost" for boarding?

### ISSUE THREAD

**Issue by t-kuester** [endless-sky/endless-sky#6337]
### Problem Description

I really like the game, but in my opinion boarding is too powerful, or comes with too little cost, meaning that you can build up a ridiculously large and powerful fleet by capturing ships in the early game (even with a Shuttle, provided that others help in the fight), enlarge your fleet, sell some for a bigger flagship, capture larger vessels, etc. All this at the very low cost of a bit of crew upkeep and boarding weapons.
In particular, I think that this encourages extremely out-of-role behavior. Once I'm in the middle of the campaign, captaining a Dreadnought in a full-out war, it's okay to send my crew to board another capital ship, but in the early game when you are just a small captain, it's just _wrong_ to send in your crew to capture another ship with "good" 80% chance of capture and accepting that half your crew will die in the process. But the game greatly encourages/rewards it.

### Related Issue Links

None that I know of.

### Desired Solution

Boarding should come with more of a "cost" or "risk", so it is not as easy to exploit, especially in the early game. Here are a few suggestions:
* introduce a compensation to be paid if one of your crew dies
* make it harder to find crew to hire; not each small planet should have hundreds of crew waiting to re-fill your fleet
* make it even harder once you have a reputation for killing off your own crew
* make one boarding weapon disappear for each crew killed
* you are part of the boarding team; each time a crew member dies in boarding, there's a 1/#crew chance that it's you
* introduce names and experience for individual crew members

### Alternative Approaches

See above.

### Additional Context

None.

**Comment by vitalchip** [comment]
Hot topic, surprised it hasn't come up before. 

**introduce a compensation to be paid if one of your crew dies**

used to be the case and was removed.

**make it harder to find crew to hire; not each small planet should have hundreds of crew waiting to re-fill your fleet**

Yes, feel the same way about most things, too often available.

**make it even harder once you have a reputation for killing off your own crew**

Yes, get a reputation as Captain death and it should be hard to recruit.

**make one boarding weapon disappear for each crew killed**

Also some should be consumed when used at all.

**you are part of the boarding team; each time a crew member dies in boarding, there's a 1/#crew chance that it's you**

Random sudden death isn't going to go down well..

**introduce names and experience for individual crew members**

Not sure about this, having different crew skills (marines) has been discussed before. Personally I don't care if they're called Pete or Polly or what their life story is.

**Comment by Chopliver19** [comment]
I agree this is an important issue, although I have to admit this is how I got a large fleet in the game.

**Comment by Chopliver19** [comment]
Option Number 4 sounds great. I can't quite wrap my head around a box of infinite grenades...

**Comment by Hector-of-the-Void** [comment]
similar to what was suggested, There could be a hiring bonus you pay when you hire, or when you hire past the needed crew number.

**Comment by yafflehk** [comment]
Maybe have different types of hire that cost more or less - bunch of spaceport bums = cheap but crap, boarding doesn't work that well. Experienced marine tactical squad with their own weapons and gear = expensive but they kick ass.

**Comment by yafflehk** [comment]
Thinking about this more as I blast my way across the galaxy in my fleet of shield beetles:
Changes to ship capturing.
A captured ship is crippled – it has no value either monetarily or as a useful ship. It can only support a minimal crew and limp back to a shipyard. In this new system all captured ships, regardless of your crew numbers, behave as though they are vastly under-crewed. A captured ship cannot fight, move or turn quickly or land on a planet without a shipyard. They will stay that way until repaired or sold.

The ship is now classed “Salvage”. As is, it has scrap or parts value – perhaps 5% of its value when new.
It can be sold for that value at any time but a ship with a salvage title will stay in unusable condition unless the player pays for it to be repaired at a shipyard – for simplicity’s sake this costs 10% of the new value.

Repair might take several weeks – depending on the size of the ship etc. Let’s say a Wasp can be repaired to full order in 3 days, a Bactrian in 3 weeks game time. A repaired ship is the same as a captured ship in the current game and can be sold or used by the player.
This doesn't solve the problem outright, but it does make capping ships a bit more time and labor intensive than it is now...

**Comment by pepoluan** [comment]
> Repair might take several weeks – depending on the size of the ship etc. Let’s say a Wasp can be repaired to full order in 3 days, a Bactrian in 3 weeks game time. A repaired ship is the same as a captured ship in the current game and can be sold or used by the player.

This is a good suggestion!
